gap/attributes/maximal.gd

```diff
@@ -1,19 +1,16 @@
 #############################################################################
 ##
 #W  maximal.gd
-#Y  Copyright (C) 2013-15                                James D. Mitchell
+#Y  Copyright (C) 2013-16                                James D. Mitchell
+##                                                       Wilfred A. Wilson
 ##
 ##  Licensing information can be found in the README file of this package.
 ##
 #############################################################################
 ##
 
-DeclareAttribute("MaximalSubsemigroups", IsSemigroup);
-DeclareOperation("MaximalSubsemigroups", [IsSemigroup, IsGroup]);
-# FIXME What does this do?
-DeclareOperation("MaximalSubsemigroupsNC", [IsReesMatrixSubsemigroup, IsGroup,
-                 IsList, IsAssociativeElement]);
-# FIXME What does this do?
-DeclareOperation("MaximalSubsemigroupsNC", [IsReesZeroMatrixSubsemigroup,
-                 IsGroup, IsDigraph, IsList, IsList, IsList]);
 DeclareOperation("IsMaximalSubsemigroup", [IsSemigroup, IsSemigroup]);
+DeclareAttribute("MaximalSubsemigroups", IsSemigroup);
+DeclareAttribute("NumberMaximalSubsemigroups", IsSemigroup);
+DeclareOperation("MaximalSubsemigroups", [IsSemigroup, IsRecord]);
```

Comment:
@james-d-mitchell I think `NrMaximalSubsemigroups` is the better name, I'll change it.